### ISSUE TITLE
The new Paradox Bags don't work with shift + e

### ISSUE THREAD

**Issue by godspeedfunnyman** [tgstation/tgstation#60023]
## Reproduction:

Wear a paradox bag in your belt then try using shift + e to take an item from it or put an item into it and it won't work

**Comment by Sheits** [comment]
on it

**Comment by JohnFulpWillard** [comment]
![image](https://user-images.githubusercontent.com/53777086/124440797-04ee2880-dd49-11eb-93a7-d76edb68f123.png)

After checking, it looks like all contents and everything, is only stored in the red paradox bag, so while the hotkey works on the red bag, it doesnt on the blue one.

**Comment by ATH1909** [comment]
it should also be noted that these bags aren't subtypes of /obj/item/storage

**Comment by Sheits** [comment]
> it should also be noted that these bags aren't subtypes of /obj/item/storage.

yeah, changing it to obj/item/storage/shared_storage let you put items into the second bag using the hotkey but not take them out
![image](https://user-images.githubusercontent.com/81964183/124457117-cef38780-ddce-11eb-94cb-c87789c84325.png)

**Comment by rdragan** [comment]
Letting these go in the belt slot was a mistake
